core/drivers/clk/clk-stm32mp15.c

```diff
@@ -353,6 +354,8 @@ static const struct stm32mp1_clk_gate stm32mp1_clk_gate[] = {
 	_CLK_FIXED(SEC, RCC_DDRITFCR, 9, DDRPHYCAPB, _PCLK4),
 	_CLK_FIXED(SEC, RCC_DDRITFCR, 10, DDRPHYCAPBLP, _PCLK4),
 
+	_CLK_SC2_SELEC(SEC, RCC_MP_APB1ENSETR, I2C5EN, I2C5_K, _I2C35_SEL),
```

Comment:
I would suggest you add I2C5 clock under a config switch `#ifdef CFG_WITH_NSEC_I2CS`, with s/`SEC`/``N_S`/ as 1st argument.